config.toml

```diff
@@ -0,0 +1,3 @@
+OPENAI_API_KEY="" 
```

Comment:
I don't think `config.toml` should be tracked in version control except as a template. Settings like which model is used, workspace dir, etc. are user-specific.

Reply:
Maybe I'm wrong on how gitignore works, but I'm pretty sure if we check this file in, changes to it won't be ignored :/

You can test by making a change to it--most likely it will show up in `git status`

Reply:
Just checked, you're right. How about we rename it to `config.toml.template` and add README instructions to copy to `config.toml`?

Then we can keep `config.toml` in .gitignore but avoid checking it in.

Reply:
👍 sgtm!